### PR TITLE
Improvements 2022

### PR DESCRIPTION
**Pull request by JohnMoehrke**
Draft Pull Request for 4.2.0 public-comment

### CODE REVIEW

File: input/images-source/usecase1-processflow.plantuml [hunk 1]
```diff
@@ -1,11 +1,12 @@
...
 hide footbox
 participant "Document Source" as DocumentSource
 participant "Document Recipient" as DocumentRecipient
-"DocumentSource" -> "DocumentRecipient" : 1. Document Source Initiates Publication Request
-activate "DocumentSource"
-activate "DocumentRecipient"
-"DocumentRecipient" --> "DocumentSource" : 2. Documents are Published
-deactivate "DocumentSource"
-deactivate "DocumentRecipient"
+DocumentSource -> DocumentRecipient : 1. Document Source Initiates Publication Request
```
Comment (oliveregger) [inline input/images-source/usecase1-processflow.plantuml:6]: Document Source initiates Publication Request

File: input/images-source/usecase1-processflow.plantuml [hunk 2]
```diff
@@ -1,11 +1,12 @@
...
+activate DocumentSource
+activate DocumentRecipient
+DocumentRecipient --> DocumentSource : 2. Documents are Published
```
Comment (oliveregger) [inline input/images-source/usecase1-processflow.plantuml:9]: Documents are published

File: input/pagecontent/31_xds.md [hunk 3]
```diff
@@ -16,6 +16,7 @@
 |------------|------------|--------------------------|
 | UnknownRecipient | An IntendedRecipient is not recognized. | P |
 | UnavailableRecipient | An IntendedRecipient which would have been contacted was not available. | P |
+| FHIRdocumentNotSupported | A request to Publish a FHIR Document was attempted but not supported by the Document Recipient Actor. | P |
```
Comment (oliveregger) [inline input/pagecontent/31_xds.md:19]: FHIRDocumentNotSupported (capitalize D?)

File: input/images-source/ITI-106-seq.plantuml [hunk 4]
```diff
@@ -0,0 +1,17 @@
+@startuml
+hide footbox
+autonumber
+participant "Document Consumer" as DocumentConsumer
+participant "Document Recipient" as DocumentRecipient
+autonumber
+activate DocumentConsumer
+activate DocumentRecipient
+DocumentConsumer -> DocumentRecipient : Generate Metadata (document)
```
Comment (oliveregger) [inline input/images-source/ITI-106-seq.plantuml:9]: This should be DocumentSource and not Document Consumer according to text: "The Generate Metadata Option recognizes that there are Document Source Actors that have a structured and coded format of a document"

Comment (JohnMoehrke) [reply input/images-source/ITI-106-seq.plantuml:9]: will fix

**Pull request merged**